nautobot/dcim/templates/dcim/powerfeed_retrieve.html

```diff
@@ -93,6 +110,7 @@
 {% endblock content_left_page %}
 
 {% block content_right_page %}
+        {% if not object.destination_panel %}
```

Comment:
Are `connected_endpoint` and `destination_panel` intended to be mutually exclusive now? If so, feels like we need to enforce that in the model.

Reply:
Need to add enforcement in `clean()` and also `save()`.